manifests/Riot/LOL/10.0.0.0.yaml

```
@@ -0,0 +1,15 @@
+Id: riot.lol
+Publisher: Riot Games Inc
+Name: League of Legend
+Version: 10.0.0.0
+License: Limited License
+InstallerType: EXE
+LicenseUrl: https://www.riotgames.com/en/terms-of-service
+AppMoniker: LOL
+Tags: LOL, League of Legend, Riot
+Description: League of Legends is a team-based strategy game where two teams of five powerful champions face off to destroy the other’s base.
+Homepage: https://leagueoflegends.com/
+Installers:
+  - Arch: x86
+    Url: https://lol.secure.dyn.riotcdn.net/channels/public/x/installer/current/live.na.exe
+    Sha256: 3F9DF64872FF68FD3ADCB4A8B6BA3082CAA72F40C11782C78703A9FE493429DB
```

```suggestion
    Sha256: 3F9DF64872FF68FD3ADCB4A8B6BA3082CAA72F40C11782C78703A9FE493429DB
    
```

GitHub does not like if there is no extra line at the end of a file